lib/tasks/incrementals.rake

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'mail'
+
+TRAJECT_HOME = '/opt/psulib_traject'.freeze
+TRAJECT_LOGS_HOME = '/var/log/traject'.freeze
+SIRSI_DATA_HOME = '/data/symphony_data'.freeze
+
+# This job, and :delete_daily expect there to be file to add and delete, it is not responsible for getting those files
+# from the catalog.
+namespace :incrementals do
+  desc 'Adds to the index'
+  task :import_daily do
+    daily_addition_files = Dir["#{SIRSI_DATA_HOME}/daily/*.mrc"]
+
+    daily_addition_files.each do |f|
+      system("bundle exec traject -s log.file=#{TRAJECT_LOGS_HOME}/traject_incremental.log -s "\
```

Comment:
@cdmo I am asking the same question (sorry), why do we have to do each here but not let traject cmd to run all the incrementals? 
`bundle exec traject ... -c #{TRAJECT_HOME}/lib/traject/psulib_config.rb #{SIRSI_DATA_HOME}/daily/*.mrc`

Reply:
To find ones that fail is what I was thinking. And if there's just one bad one, then at least you're processing the other stuff. But, yeah, it would be simpler to just do it this way. Chances are we'll catch the error quickly and fix it. I think your suggestion is the better solution. This has been merged already, so we can a quickfix branch kind of thing